packages/terra-clinical-label-value-view/lib/LabelValueView.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
+
+.terraClinical-LabelValueView {
+  border: 0;
+  padding: 0.1785rem;
+}
+
+.terraClinical-LabelValueView-label {
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 20px;
+}
+
+.terraClinical-LabelValueView-value {
+  font-size: 1.4285714285714286rem;
+  line-height: 1.4285714285714286rem;
```

Comment:
We could do two separate props instead of children. `valueText` and `valueElement` or `valueComponent` or something. I'd (personally) like to avoid type-checking props. Or we could still use children too, something like:

```
<div {...customProps} className={labelValueViewClassNames}>
  <div className="terraClinical-LabelValueView-label">{props.label}</div>
  <div className="terraClinical-LabelValueView-value">{props.valueText}</div>
  {children}
</div>
```
We'd just need to document that valueText will be styled a certain way, and that children could be used for more substantial components.